martin/src/pg/pool.rs

```diff
@@ -42,21 +45,26 @@ impl PgPool {
 
         let conn = get_conn(&pool, &id).await?;
         let pg_ver = get_postgres_version(&conn).await?;
-        if pg_ver < MINIMUM_POSTGRES_VER {
-            return Err(PostgresqlTooOld(pg_ver, MINIMUM_POSTGRES_VER));
+        if pg_ver < MINIMUM_POSTGRES_VERSION {
+            return Err(PostgresqlTooOld(pg_ver, MINIMUM_POSTGRES_VERSION));
         }
-        if pg_ver < RECOMMENDED_POSTGRES_VER {
-            warn!("PostgreSQL {pg_ver} is older than the recommended {RECOMMENDED_POSTGRES_VER}.");
+        if pg_ver < RECOMMENDED_POSTGRES_VERSION {
+            warn!("PostgreSQL {pg_ver} is older than the recommended {RECOMMENDED_POSTGRES_VERSION}.");
         }
 
         let postgis_ver = get_postgis_version(&conn).await?;
-        if postgis_ver < MINIMUM_POSTGIS_VER {
-            return Err(PostgisTooOld(postgis_ver, MINIMUM_POSTGIS_VER));
+        if postgis_ver < MINIMUM_POSTGIS_VERSION {
+            return Err(PostgisTooOld(postgis_ver, MINIMUM_POSTGIS_VERSION));
         }
-        let margin = postgis_ver >= RECOMMENDED_POSTGIS_VER;
-        if !margin {
-            warn!("PostGIS {postgis_ver} is older than the recommended {RECOMMENDED_POSTGIS_VER}. Margin parameter in ST_TileEnvelope is not supported, so tiles may be cut off at the edges.");
+        let margin_not_supported = postgis_ver < ST_TILE_ENVELOPE_POSTGIS_VERSION;
+        if margin_not_supported {
+            warn!("PostGIS {postgis_ver} is older than {ST_TILE_ENVELOPE_POSTGIS_VERSION}. Margin parameter in ST_TileEnvelope is not supported, so tiles may be cut off at the edges.");
         }
+        let tiles_could_be_missing_geometry = postgis_ver < MISSING_GEOM_FIXED_POSTGIS_VERSION;
+        if !tiles_could_be_missing_geometry {
```

Comment:
i think you meant this without the `!`
```suggestion
        if tiles_could_be_missing_geometry {
```